src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar
+
+data class Car(val name: String, var position: Int = 0) {
```

Comment:
이렇게 매개변수를 줄바꿈해주는 방식이 코틀린 컨벤션에 조끔더 적합하지 않을까?🤔 생각해봅니다!
```suggestion
data class Car(
    val name: String,
    var position: Int = 0,
) {
```

Reply:
2가지 방법 모두 코틀린 컨벤션에 부합한데, 팀의 스타일 가이드에 따라 다른 걸로 알고 있습니다! 저는 파라미터가 짧고 간결한 경우 한 줄 작성이 권장된다고 알고 있어서 이렇게 해봤습니다.